### DISCUSSION TITLE
Should we allow for inplace operations in fg?

### DISCUSSION THREAD

**Discussion by ferrine** [pymc-devs/pytensor#74] [General]
https://github.com/pymc-devs/pytensor/blob/d78e1a1ad7b4b2f4d13d0c3043ea79ca53aa24d0/pytensor/graph/fg.py#L445

**Comment by ricardoV94** [comment]
This is fundamentally how graph rewrites work: https://gist.github.com/ricardoV94/deaf7b18660588faac1c30bc5b31c011
